setup.py

```diff
@@ -18,14 +18,12 @@
 tests_require = [
     'check-manifest>=0.25',
     'coverage>=4.0',
-    # FIXME: Remove elasticsearch once pytest-invenio have fixed the es import.
-    'elasticsearch>=5.0.0',
-    'elasticsearch-dsl>=5.0.0',
+    'urllib3>=1.21.1,<1.25',
     'invenio-db>=1.0.0b3',
     'isort>=4.3.4',
     'pydocstyle>=1.0.0',
-    'pytest-cov>=1.8.0',
-    'pytest-invenio>=1.0.0',
+    'pytest-cov>=2.7.1',  # FIXME: moved to pytest-invenio
```

Comment:
Is this `FIXME` a forgotten one, or a gentle reminder for the next lucky one?

Reply:
It should be fixed after https://github.com/inveniosoftware/pytest-invenio/pull/33